elide-core/src/test/java/com/yahoo/elide/core/EntityDictionaryTest.java

```diff
@@ -58,4 +64,40 @@ public void testGetInverseRelationshipOwnedSide()  {
         Assert.assertEquals(getRelationInverse(Child.class, "parents"), "children",
                 "The inverse relationship of children should be parents");
     }
+
+    @Test
+    public void testGetIdAnnotations() throws Exception {
+
+        Collection<Class> expectedAnnotationClasses = Arrays.asList(new Class[]{Id.class, GeneratedValue.class});
+        Collection<Class> actualAnnotationsClasses  = getIdAnnotations(new Parent()).stream()
+            .map(Annotation::annotationType)
+        .collect(Collectors.toList());
```

Comment:
This style was intentional. Slightly modeled after patterns seen in JS. `stream` and `collect` are the book ends and the `map` is the meat. Are you saying `map` should not have an indent?